book/website/community-handbook/presenting.md

```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
+### Planning a Talk
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
+
+### Preparing Slides
+Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.
+
+### Giving a Talk
+We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.
+
+### After Your Talk
+Keep spreading the word! Please save your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```

Comment:
```suggestion
Keep spreading the word! Please archive your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.

Please make sure that you credit all of the contributors and presenters of your presentation as authors in the Zenodo upload form. If possible, link to any previous talks where you have used additional slides outside of the provided _The Turing Way_ presentation templates - this can be done in the description part of the upload form.
```

Reply:
```suggestion
Keep spreading the word! Please archive your slides in two formats: their original (editable) format and as a PDF. If you don't know how to convert PowerPoint slides to PDF, see these [instructions](https://www.wikihow.com/Convert-Powerpoint-to-PDF); following these instruction helps preserve accessibility features for people using screen readers. You can archive your slides on [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```